Makefile

```diff
@@ -185,12 +185,18 @@ $(LEGATO_SOURCES_FETCHED):
 	# 49737 = size reduction by removing curl, zlib and openssl from apps.
 	# 49773 = make symlinks in the bin directory relative instead of absolute.
 	# 50657 = remove excessive warning messages in syslog due to failure to open an NMEA pipe
+	# 50868 and 50869 = fix memory leak and permanently busy state in COAP layer needed by Octave.
 	cd $(LEGATO_ROOT) && \
 		git fetch ssh://gerrit.legato:29418/Legato refs/changes/37/49737/1 && \
 		git cherry-pick FETCH_HEAD && \
 		git fetch ssh://gerrit.legato:29418/Legato refs/changes/73/49773/1 && \
 		git cherry-pick FETCH_HEAD && \
 		git fetch ssh://gerrit.legato:29418/Legato refs/changes/57/50657/1 && \
+		git cherry-pick FETCH_HEAD && \
+		git fetch ssh://gerrit.legato:29418/Legato refs/changes/68/50868/5 && \
+		git cherry-pick FETCH_HEAD && \
+		cd apps/platformServices/airVantageConnector && \
+		git fetch ssh://gerrit.legato:29418/Legato/Service/AirVantageConnector refs/changes/69/50869/9 && \
```

Comment:
Do we have confirmation from the Octave team that these are the only additional patched that we need?

Reply:
No.  Actually, we know they bring in a bunch more.  The list has been changing rapidly lately, as they prepare for their GA release.